### PR TITLE
Full json

### PR DESCRIPTION
**Pull request by sean-mccorkle**
added chunking to src/utils/import_json_file.py
added newlines to document output in src/modelseed/generate_reaction_similarities_json.py

### CODE REVIEW

File: src/modelseed/generate_reaction_similarities_json.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    if len(sys.argv) < 2:
+        sys.stderr.write('Provide input file path of a csv containing similarity data and output json file path\n')
+        exit(1)
+
+    in_file_path = os.path.abspath(sys.argv[1])
+    out_json_file_path = os.path.abspath(sys.argv[2])
+    if not os.path.exists( in_file_path ):
+        sys.stderr.write('%s does not exist\n' % file_path)
+        exit(1)
+    result = convert_similarities( in_file_path, out_json_file_path )
```
Comment (jayrbolton) [inline src/modelseed/generate_reaction_similarities_json.py:46]: To remove the lgtm.com warning, remove the `result = ` here.

File: src/utils/import_json_file.py [hunk 2]
```diff
@@ -15,16 +15,20 @@
 import requests
 import logging
 import argparse
+import json
```
Comment (jayrbolton) [inline src/utils/import_json_file.py:18]: To remove the lgtm.com warning, remove `import json` here

**Comment by jayrbolton** [comment]
I guess we should try and please that lgtm-com bot and remove those alerts. I commented on the lines where those alerts occur.

**Comment by sean-mccorkle** [comment]
Yeah I wasn't sure if it was worth it or not - will take that advice from now on and clear them up

**Comment by jayrbolton** [comment]
Content looks good to me :+1:

**Comment by sean-mccorkle** [comment]
Looks like lgtm is happy now

**Comment by jayrbolton** [comment]
All merged

**Pull request merged**